### PR TITLE
[AKS] Add support for SGX addon 'confcom'

### PR DESCRIPTION
**Pull request by Pengpeng-Microsoft**
**Description**

Add support for SGX addon 'confcom'. AKS cluster created with confcom addon enabled will have support with SGX device plugin for Gen2 VMs.

**Testing Guide**

Add tests for: a) AKS cluster creation with confcom addon enabled with default option; b) AKS cluster creation with confcom addon enabled with SGX quote helper option; c) enablement of confcom addon for an AKS cluster; d) disablement of confcom addon for an AKS cluster.

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/databoxedge/tests/latest/test_device_scenario_coverage.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 |Scenario|Result|ErrorMessage|ErrorStack|ErrorNormalized|StartDt|EndDt|
-|step_device_create|successed||||2021-01-13 08:31:38.209522|2021-01-13 08:31:43.981223|
-|step_device_show|successed||||2021-01-13 08:31:56.811539|2021-01-13 08:31:59.056123|
-|step_device_list|successed||||2021-01-13 08:32:01.747769|2021-01-13 08:32:05.003301|
-|step_device_list2|successed||||2021-01-13 08:31:52.250896|2021-01-13 08:31:54.480896|
-|step_device_update|successed||||2021-01-13 08:31:54.481895|2021-01-13 08:31:56.810543|
-|step_device_delete|successed||||2021-01-13 08:31:59.068123|2021-01-13 08:32:01.747769|
+|step_device_create|successed||||2021-02-09 05:32:14.712695|2021-02-09 05:32:15.848727|
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/databoxedge/tests/latest/test_device_scenario_coverage.md:2]: just curious, why need change databoxedge file?

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -1892,6 +1893,7 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
                appgw_id=None,
                appgw_subnet_id=None,
                appgw_watch_namespace=None,
+               enable_sgxquotehelper=False,
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py:1896]: default is false or None?

Comment (Pengpeng-Microsoft) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py:1896]: Default is false.

File: src/azure-cli/azure/cli/command_modules/databoxedge/tests/latest/test_device_scenario_coverage.md (hunk 1 above)
Comment (Pengpeng-Microsoft) [reply src/azure-cli/azure/cli/command_modules/databoxedge/tests/latest/test_device_scenario_coverage.md]: This was automatically generated by adev test. Just reverted these unrelated changes.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**